server_setup.md

```diff
@@ -0,0 +1,29 @@
+# Server Setup for creating a low privilege user
+#### Aim is to create such a user role that has the minimum privileges required to create a patient via an API call only.
+Please follow the steps listed below in order
+1. Log in to the server as an Admin.
+   * **username : admin**
+   * **password : Admin123**
+2. Go to System Adimistration -> Advanced Administartion.
+3. Under the Users section open Manage Roles -> Add Role.
+4. A form appears which asks for the following entries. Fill them with the provided values <br>
+     
+     * **Role**         :  telemedicine low privilege user
+     * **Description**  :  a low privilege role that can only register patients via API calls
+     * **Inherited Roles** : leave all boxes unchecked
+     * **Privileges** : Check the privileges listed below
+         * Add Patients
+         * App: patient list
+         * Get Patients
```

Comment:
Will this privilege allow the user you created to view patient information (via the api or otherwise)? If so, that may be problematic. We don't want malicious users of the app to be able to pull down other patients' private info.